evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md

```diff
@@ -0,0 +1,93 @@
+# Evidence-0001: Retention at to Rank 1
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission 2024/11/14                                                               |
+| **Submitted by**| George Pisaltu                                                                              |
+
+
+## Member details
+
+- Matrix username: `@george.pisaltu:parity.io`
+- Polkadot address: `128pmEUBSGjGeXZXNaAmomAJgVn77L74YT7Zdjd3fP63HWNP`
+- Current rank: `1`
+- Date of initial induction: `2024/09/09`
```

Comment:
It should read: 2024/08/16 as per [this](https://collectives-polkadot.subscan.io/extrinsic/4441984-2).

Reply:
Fixed.